tests/token_classes.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+from pydantic.class_validators import validator
+from pydantic.main import BaseModel
+
+from spylib.token import (
+    OfflineTokenABC,
+    OfflineTokenResponse,
+    OnlineTokenABC,
+    OnlineTokenResponse,
+)
+
+online_token_data = OnlineTokenResponse(
+    access_token='ONLINETOKEN',
+    scope=','.join(['write_products', 'read_customers']),
+    expires_in=86399,
+    associated_user_scope=','.join(['read_products']),
+    associated_user={
+        'id': 902541635,
+        'first_name': 'John',
+        'last_name': 'Smith',
+        'email': 'john@example.com',
+        'email_verified': True,
+        'account_owner': True,
+        'locale': 'en',
+        'collaborator': False,
+    },
+)
+
+offline_token_data = OfflineTokenResponse(
+    access_token='OFFLINETOKEN',
+    scope=','.join(['write_products', 'read_customers']),
+)
+
+
+class MockHTTPResponse(BaseModel):
+    status_code: int
+    jsondata: dict
+    headers: dict = None  # type: ignore
+
+    @validator('headers', pre=True, always=True)
+    def set_id(cls, fld):
+        return fld or {'X-Shopify-Shop-Api-Call-Limit': '39/40'}
+
+    def json(self):
+        return self.jsondata
+
+
+class AppInformation(BaseModel):
```

Comment:
Why is that called "AppInformation" and not "StoreInformation"?

Reply:
It (mostly) contains information specific to the application not the store. The `store_name` and `api_version` property is there because I needed somewhere to put a constant that could be shared across all tests. It could be put somewhere more descriptive if you'd like.

Reply:
Ok so `TestInformation`?
Sorry the current name just really threw me off when I first saw it.